### PR TITLE
Fix for table names being obscured when changing table structure

### LINKED ISSUE #13748: Table name hides column descriptions when creating table
### Steps to reproduce
1. Open database
2. Enter name of table to create and go to table creation

### Expected behaviour
The column names should be shown

screenshot-2017-10-16 localhost localhost baz3 hokus phpmyadmin 4 7 5-dev

### Actual behaviour
Column names are hidden behind table name input.

screenshot-2017-10-16 localhost localhost baz3 hokus phpmyadmin 4 8 0-dev

### Server configuration
**phpMyAdmin version:** master at 35e15e90c1, this is regression from QA_4_7

### Client configuration
**Browser:** Firefox

### PR DESCRIPTION
**Pull request by ankitjain28may**
Signed-off-by: Ankit Jain <ankitjain28may77@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: js/config.js [hunk 1]
```diff
@@ -612,7 +612,7 @@ function adjustPrefsNotification () {
     var $prefsAutoShowing = ($prefsAutoLoad.css('display') !== 'none');
 
     if ($prefsAutoShowing && $tableNameControl.length) {
-        $tableNameControl.css('top', '120px');
+        $tableNameControl.css('top', '55px');
```
Comment (nijel) [inline js/config.js]: IMHO this code is not active in my case - the preferences loading box is not present....

Comment (ankitjain28may) [reply js/config.js]: This function is triggred and it adds the inline styling i.e top = 120px to the div which creates this bug. I don't know through which its triggering but this is the cause for the bug.

**Comment by ankitjain28may** [comment]
@OlafvdSpek Check out the css.

File: js/config.js (hunk 1 above)
Comment (ankitjain28may) [reply js/config.js]: @nijel Any other suggestions ?

Comment (nijel) [reply js/config.js]: Apparently I've misunderstood the code, the fix works fine.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**